notebook_connector/secret_store.py

```diff
@@ -0,0 +1,161 @@
+import contextlib
+import logging
+import os
+from dataclasses import dataclass
+from sqlcipher3 import dbapi2 as sqlcipher
+from typing import List, Optional, Union
+from inspect import cleandoc
+
+
+_logger = logging.getLogger(__name__)
+
+
+@dataclass(frozen=True)
+class Table:
+    name: str
+    columns: List[str]
+
+
+SECRETS_TABLE = Table("secrets", ["user", "password"])
```

Comment:
I think, we have not only have user and password in the worst case. Maybe, we should define a Secret as a Key, Value pair and you can use two key value pairs for user and password,

(Can be done in a later PR)